cmd/fyne/internal/commands/get.go

```diff
@@ -57,20 +60,36 @@ func NewGetter() *Getter {
 
 // Get automates the download and install of a named GUI app package.
 func (g *Getter) Get(pkg string) error {
-	cmd := execabs.Command("go", "get", "-u", "-d", pkg)
-	cmd.Env = append(os.Environ(), "GO111MODULE=off") // cache the downloaded code
+	name := filepath.Base(pkg)
+	path := filepath.Join(os.TempDir(), name)
+	if util.Exists(path) {
+		_ = os.Remove(path)
```

Comment:
See the generic review comment for background. Is it not better to do `os.CreateTemp()` to get a fresh random temp directory and then delete it afterwards instead?

Reply:
CreateTemp creates files not directories

Reply:
I think that works nicely, thanks.